WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsViewModel.kt

```diff
@@ -292,7 +280,7 @@ class StatsViewModel
 
         listUseCases[statsSection]?.onListSelected()
 
-        if (statsSection == StatsSection.INSIGHTS) showInsightsUpdateAlert()
+        if (BuildConfig.IS_JETPACK_APP) updateRevampedInsights()
```

Comment:
I think we don't need to `updateRevampedInsights` here if we always update in `start` without checking the current `statsSection`.